tests/controllers/run_controller_tests.sh

```diff
@@ -1,16 +1,14 @@
 #!/bin/bash
 
-ENVTEST_K8S_VERSION=${ENVTEST_K8S_VERSION-1.28.3}
```

Comment:
How can we provide the k8s version now? Would it make sense to add it as an env var in the github workflow? Like we do for [fleet](https://github.com/rancher/fleet/blob/main/.github/workflows/ci.yml#L39)

Reply:
I re-added the environment variable in the script, and specified it in the workflow. I also added a line to the README on how to use it.